libs/libuwsc/Makefile

```diff
@@ -31,39 +31,48 @@ define Package/libuwsc/default
   SECTION:=libs
   CATEGORY:=Libraries
   SUBMENU:=Networking
-  TITLE:=Lightweight WebSocket client library
-  DEPENDS:=+libubox
+  TITLE:=Lightweight WebSocket client library based on libev
+  DEPENDS:=+libev
+endef
+
+define Package/libuwsc-mbedtls
+  $(Package/libuwsc/default)
+  VARIANT:=mbedtls
+  TITLE += (mbedtls)
+  DEPENDS += +libmbedtls
+  CONFLICTS:=libuwsc-nossl libuwsc-openssl libuwsc-wolfssl
 endef
 
 define Package/libuwsc-nossl
   $(Package/libuwsc/default)
-  TITLE += (NO SSL)
   VARIANT:=nossl
+  TITLE += (NO SSL)
+  CONFLICTS:=libuwsc-openssl libuwsc-wolfssl
 endef
 
 define Package/libuwsc-openssl
   $(Package/libuwsc/default)
   TITLE += (openssl)
-  DEPENDS += +libustream-openssl
   VARIANT:=openssl
+  DEPENDS += +libopenssl
+  CONFLICTS:=libuwsc-wolfssl
```

Comment:
Maybe it's okay to leave out these `CONFLICTS` specs.  Build variants of the same kind are expected to in conflict with each other anyway.  I think this is also how variants of dnsmasq, openvpn handles the situation.

Reply:
Done.